src/Nethermind/Nethermind.Evm/ReleaseSpecExtensions.cs

```diff
@@ -97,5 +97,10 @@ public static class ReleaseSpecExtensions
             spec.UseExpDDosProtection
                 ? GasCostOf.ExpByteEip160
                 : GasCostOf.ExpByte;
+
+        public static long GetTLoadCost(this IReleaseSpec spec) =>
+            GasCostOf.TLoad;
+        public static long GetTStoreCost(this IReleaseSpec spec) =>
+            GasCostOf.TStore;
```

Comment:
Not sure if needed as there is no logic here, we can just use GasCostOf directly